injected/integration-test/duckplayer.e2e.spec.js

```diff
@@ -47,7 +47,7 @@ test.describe('e2e: Duck Player Thumbnail Overlays on YouTube.com', () => {
         test('settings can change to disabled', async ({ page }, workerInfo) => {
             const overlays = DuckplayerOverlays.create(page, workerInfo);
             await overlays.withRemoteConfig({ json: 'overlays-live.json' });
-            await overlays.userSettingIs('enabled');
```

Comment:
The ordering here is a little confusing, should it happen before `withRemoteConfig` or not?

Reply:
yes it should, in these cases it's not causing a failure, but I'll alter to be consistent everywhere